get_latest_configs.sh

```diff
@@ -0,0 +1,14 @@
+#bin/bash
```

Comment:
Missing the `!` for `#!/bin/bash`, also it'd be nice to just use POSIX shell (`/bin/sh`) in case someone is using zsh instead of bash -- looks like this script doesn't do anything super fancy so we shouldn't need to worry about bash-specific features.

Reply:
Updated to use ```#/bin/sh```